packages/frontend/src/components/Button/style.module.scss

```diff
@@ -30,34 +30,38 @@
   &.primary {
     background-color: var(--buttonPrimaryBackground);
     color: var(--buttonPrimaryText);
+    border: var(--buttonPrimaryBackground);
     &:hover {
       background-color: var(--buttonPrimaryHover);
     }
   }
 
-  &.secondary {
-    background-color: var(--buttonSecondaryBackground);
-    color: var(--buttonSecondaryText);
-    border: 1px solid #ccc;
-    &:hover {
-      background-color: var(--buttonSecondaryHover);
-    }
-  }
-
   &.danger {
     color: var(--buttonDangerText);
     background-color: var(--buttonDangerBackground);
+    border: var(--buttonPrimaryBackground);
     &:hover {
       background-color: var(--buttonDangerHover);
     }
   }
 
+  &.borderless {
+    background-color: transparent;
+    border: none;
+    &:hover {
+      background-color: transparent;
+    }
```

Comment:
```suggestion
```

This removes hover color from the buttons, e.g. the archive button. I don't think we want that?